src/python/pants/engine/build_files.py

```diff
@@ -222,7 +227,7 @@ def _hydrate(item_type, spec_path, **kwargs):
   address_family_by_directory = {af.namespace: af for af in address_families}
 
   matched_addresses = OrderedSet()
-  addr_to_origin: Dict[BuildFileAddress, AddressSpec] = {}
+  bfaddr_to_origin: Dict[BuildFileAddress, AddressSpec] = {}
```

Comment:
The below changes are only renames to make clear that this is a `BFA` rather than `Address`. Will make the follow up easier to understand.